appveyor.yml

```
@@ -1,7 +1,7 @@
 version: 1.0.{build}
 image: Windows Server 2019
 services:
-  - postgresql101
+  - postgresql111

```

This is incorrect to my knowledge. Should be:
```suggestion
  - postgresql11

```

---

But rebasing from the latest will bring pg 12, so it will not be an issue.